feat(account-lib): add support for ecdsa keyshare generation tss

sachushajiabraham opened the pull request:
this change add support for ecdsa signature key share generation for TSS

TICKET: BG-48875

wesgraham commented on modules/account-lib/src/mpc/tss/ecdsa/ecdsa.ts:
```diff
@@ -0,0 +1,90 @@
+  static shamir: Shamir = new Shamir(Ecdsa.curve);
+  /**
+   * Generate shares for player at index and split keys `(t,n)` ways.
+   * @param {Number} index Player index
```
where does the term "player" come from? can we use "participant" in our naming to be consistent with docs/other implementations

mmcshinsky-bitgo commented on modules/account-lib/src/mpc/tss/ecdsa/ecdsa.ts:
```diff
@@ -0,0 +1,90 @@
+  static shamir: Shamir = new Shamir(Ecdsa.curve);
+  /**
+   * Generate shares for participant at index and split keys `(threshold,numShares)` ways.
+   * @param {Number} index participant index
```
Inconsistent doc casing with method params.

sachushajiabraham commented on modules/account-lib/src/mpc/util.ts:
```diff
@@ -30,6 +30,10 @@ export function bigIntToBufferBE(n: bigint, bytes?: number): Buffer {
   return buf;
 }
 
+export function bigIntToHex(a: bigint): string {
```
Yup. Updated.

OttoAllmendinger commented:
Why was @BitGo/btc-team added as reviewer? Removed it.

sachushaji committed:
feat(account-lib): add support for ecdsa keyshare generation tss

this change add support for ecdsa signature key share generation for TSS

TICKET: BG-48875

ducnguyen-bitgo reviewed (approved):
Lgtm.

sachushajiabraham commented:
@mmcshinsky-bitgo can you kindly re check this PR

olibeyene commented on modules/sdk-core/src/account-lib/mpc/index.ts:
```diff
@@ -1,8 +1,11 @@
 export * from './util';
 
-export { Eddsa, Ed25519BIP32, HDTree, KeyShare };
+type KeyShare = EDDSA.KeyShare;
```
can ecdsa and eddsa share KeyShare type?

sachushajiabraham replied:
Nope , Left Keyshare export  like this to be backward compatible.
